cmake/CMakeConfig.cmake

```diff
@@ -0,0 +1,2 @@
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+set(CMAKE_VERBOSE_MAKEFILE OFF)
```

Comment:
newline at the EOF <3
Also, project() defaults CMAKE_VERBOSE_MAKEFILE to FALSE as per https://cmake.org/cmake/help/latest/variable/CMAKE_VERBOSE_MAKEFILE.html